pkg/workloads/cortex/lib/server/tensorflow.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2020 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import grpc
+import time
+
+from tensorflow_serving.apis import model_service_pb2_grpc
+from tensorflow_serving.apis import model_management_pb2
+from tensorflow_serving.config import model_server_config_pb2
+
+from cortex.lib.exceptions import CortexException
+from cortex.lib.log import cx_logger
+
+
+class TensorFlowServing:
+    def __init__(self, address):
+        self.address = address
+        self.model_platform = "tensorflow"
+        self.channel = grpc.insecure_channel(self.address)
+        self.stub = model_service_pb2_grpc.ModelServiceStub(self.channel)
+        self.timeout = 900
```

Comment:
Is this timeout in seconds or milliseconds? If it is seconds, it looks like the timeout is 15 minutes to get a response for HandleReloadConfigRequest. I am not sure how long this request should take. Why not 10 seconds? We could catch the timeout and retry.

Reply:
I've seen this thing need even 60-90 seconds. And this was just for a model (unfortunately, I can't remember which one it was). Basically, the request won't return until all models are loaded, which if they are a handful (> 10) it might take some time. I set `replace_models` to `False` (where it's called) so that it uses the `MergeFrom` function instead. Using the `MergeFrom` function instead of `CopyFrom` won't discard the already loaded models if there's a new request after a failed one.